Content.Shared/_Harmony/CCVars/HCCVars.cs

```diff
@@ -0,0 +1,23 @@
+using Robust.Shared.Configuration;
+
+namespace Content.Shared._Harmony.CCVars;
+
+/// <summary>
+/// Harmony-specific cvars.
+/// </summary>
+[CVarDefs]
+public sealed class HCCVars
+{
+    /// <summary>
+    /// Modifies suicide command to ghost without killing the entity.
+    /// </summary>
+    /// Frontier: Disabled the anti suicide code pending discussion about wanting it
+   /// public static readonly CVarDef<bool> DisableSuicide =
+   ///     CVarDef.Create("ic.disable_suicide", false, CVar.SERVER);
+   /// end Frontier
```

Comment:
for instances like this, its sufficient to just disable it within the active cvar itself instead of commenting out the code. This allows our downstreams/server hosts to decide for themselves if they want the functionality or not, while still keepinig it "off" for us. Personally, im not 100% understanding the complications regarding suicide actions killing the host mob entity, and am open to discuss the thought process behind why (i understand you also probably dont have a strong opinion either lol)